topics/microbiome/tutorials/dada2/tutorial.md

```diff
@@ -0,0 +1,884 @@
+
+> <question-title></question-title>
+>
+> 1. Are the estimated error rates (black line) in a good fit to the observed rates (points) for the forward reads?
+> 2. What about the reverse reads?
+>
+>    ![Error rates for each possible transition as a function of the associated quality score, the final estimated error rates, the initial input rates, and the expected error rates under the nominal definition of quality scores](./images/learnErrors_reverse.png "Error rates for each possible transition for the reverse reads")
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The estimated error rates (black line) are a good fit to the observed rates (points), and the error rates drop with increased quality as expected. Everything looks reasonable and we proceed with confidence.
+> > 2. Same
+> >
+> {: .solution}
+>
+{: .question}
+ 
+
+## Infer Sample Composition 
+
+In this step, the core sample inference algorithm ({% cite Callahan_2016 %}) is applied to the filtered and trimmed sequence data.
+
+<div class="Step-by-step" markdown="1">
+
+> <hands-on-title> Sample Inference </hands-on-title>
+>
+> 1. {% tool [dada2: dada](toolshed.g2.bx.psu.edu/repos/iuc/dada2_dada/dada2_dada/1.28+galaxy0) %} with the following parameters:
+>    - *"Process samples in batches"*: `no`
+>        - {% icon param-file %} *"Reads"*: `forward` output of **Unzip collection**
+>        - *"Pool samples"*: `process samples individually`
+>
+>           > <comment-title>Sample pooling</comment-title>
+>           >
+>           > By default, the **dada2: dada** function processes each sample independently. However, pooling information across samples can increase sensitivity to sequence variants that may be present at very low frequencies in multiple samples. Two types of pooling are possible:
+>           >
+>           > - `poll sample`: standard pooled processing, in which all samples are pooled together for sample inference
+>           > - `pseudo pooling between individually processed samples`: pseudo-pooling, in which samples are processed independently after sharing information between samples, approximating pooled sample inference in linear time.
+>           >
+>           {: .comment}
+>
+>    - {% icon param-file %} *"Error rates"*: `errors for forward reads`
+>
+> 2. Rename output collection to `sample inference for forward reads`
+>
+>    {% snippet faqs/galaxy/collections_rename.md %}
+>
+> 3. {% tool [dada2: dada](toolshed.g2.bx.psu.edu/repos/iuc/dada2_dada/dada2_dada/1.28+galaxy0) %} with the following parameters:
+>    - *"Process samples in batches"*: `no`
+>        - {% icon param-file %} *"Reads"*: `reverse` output of **Unzip collection**
+>        - *"Pool samples"*: `process samples individually`
+>    - {% icon param-file %} *"Error rates"*: `errors for reverse reads`
+>
+> 4. Rename output collection to `sample inference for reverse reads`
+{: .hands_on}
+
```

Comment:
We could add that the `"Process samples in batches"`: ` yes` is equivalent to the default setting (i.e. no pooling). 

But samples will be processed in different jobs that might run in parallel (instead of sequentially in one job).